stdlib/rubygems/0/rubygems.rbs

```diff
@@ -99,6 +94,10 @@ end
 # -The RubyGems Team
 #
 module Gem
+  interface _HashLike[K, V]
+    def each_pair: () { ([ K, V ]) -> untyped } -> self
+  end
+
```

Comment:
[note] According to this advice: https://github.com/ruby/rbs/pull/605#discussion_r581879772